src/simplewallet/simplewallet.cpp

```diff
@@ -4048,9 +4048,9 @@ bool simple_wallet::ask_wallet_create_if_needed()
  */
 void simple_wallet::print_seed(const epee::wipeable_string &seed)
 {
-  success_msg_writer(true) << "\n" << boost::format(tr("NOTE: the following %s can be used to recover access to your wallet. "
-    "Write them down and store them somewhere safe and secure. Please do not store them in "
-    "your email or on file storage services outside of your immediate control.\n")) % (m_wallet->multisig() ? tr("string") : tr("25 words"));
+  success_msg_writer(true) << "\n" << boost::format(tr("NOTE: the following %s can be used to recover access to your wallet.\n"
+    "Write them down and store them somewhere safe and secure. Please do not store\n"
```

Comment:
Why does this need a line break in the middle of the sentence, should wrap automatically depending on size?

Just thinking, I've seen a bunch of other instances where line length is long, but without breaks, and maybe there should be a consistent standard throughout. Example:

https://github.com/monero-project/monero/blob/master/src/simplewallet/simplewallet.cpp#L5819-L5822

Reply:
It does not wrap automatically.

![wrap](https://user-images.githubusercontent.com/101352116/183529040-7ddc873f-6268-4f55-a880-ef3cba27b689.jpg)